encrypted-media-respec.html

```diff
@@ -600,6 +600,13 @@
             multiple independent profiles for a single account.
           </p>
         </dd>
+        <dt id="valid-media-mime-type">Valid Media MIME Type</dt>
+        <dd>
+          <p>
+            A media <a def-id="mime-types">MIME type</a> [[!HTML51]] represents the type of the <a def-id="media-resource">media resource</a>. Its MUST be a <a def-id="valid-mime-type">valid media MIME type</a> [[!HTML51]]. 
```

Comment:
I meant for "valid-media-mime-type" to point at the definition here, and "valid-mime-type" to link to the HTML5 definition.  I inserted "media" too many places, and will make it more consistent.